style.css

```diff
@@ -0,0 +1,138 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;0,700;1,700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+#root {
+  background-color: #f1f5fa;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: white;
+  text-decoration: none;
+}
+
+.navbar {
+  width: 100%;
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: space-between;
+  width: 350px;
+  list-style: none;
+}
+
+.nav-items li a {
+  display: inline-block;
+  padding: 15px;
+  font-size: 16px;
+  line-height: 34px;
+  color: #fff;
+}
+
+.nav-items li {
+  color: blue;
+}
+
+.navbar .openMenu {
+  font-size: 1.8rem;
+  margin: 10px;
+  font-weight: 100;
+  color: #7f8cff;
+  display: none;
+  cursor: pointer;
+}
+
+.navbar #logo img {
+  width: 100px;
+  margin: auto;
+}
+
+#index {
+  top: 0;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 110vh;
+  background-image: url(./images/desktop-images/deskBackground-Home.png);
+  margin-bottom: 230px;
+  background-position-y: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] I must be sincere you have done great work designing your portfolio project, but i would like to highlight some changes below that you need to work on to make your design looks great and perfect with what we have in the Figma design.

  - I would kindly suggest you work on the background image and be sure its appearing just like what we have in the Figma design
  - On the id `#index`, kindly change your `height` to `100vh`, remove the `margin-bottom` and the `background-position-y`
  - On the `#index #container-def` styling, kindly change the `width` to `85%` and the `height` to `50%`
  - On the id `#root`, kindly change the `background-color` to `#e5e5e5` as this match with the color on the Figma design
  - While the Background-color is changed, you need to give your `header` a `class` or an `id`, then give the `class name` or `id name` a `background-color` of `#fff`.
  
With the above changes highlighted, I believe your work will be perfect and ready for approval. Kindly work on it and let me know if the is anywhere you are not clear about.

Your Design 👇 

![obong](https://user-images.githubusercontent.com/78634841/219589555-bd295e4c-4430-425d-b303-348246c70fd3.png)

Figma Design 👇 

![real](https://user-images.githubusercontent.com/78634841/219589883-919a72b9-0d06-4127-868a-f31a57e228e8.png)